docs/public/operator-hub.adoc

```diff
@@ -1,271 +1,151 @@
 
-odo provides the feature to print the YAML definition of the service (Custom
-Resource or CR) provided by the Operator before starting a service off it. This
-can be done by:
+=== Listing all available Operators
 
-[source,shell]
+Before deploying your first Operator, have a look at what is available:
+[source,sh]
 ----
-$ odo service create <operator-name> --crd <cr-name> --dry-run
+  $ odo catalog list services
+  Operators available in the cluster
+  NAME                          CRDs
+  etcdoperator.v0.9.4           EtcdCluster, EtcdBackup, EtcdRestore
 ----
 
-For example, to print YAML definition of `EtcdCluster` provided by
-`etcdoperator.v0.9.4` Operator, you would do:
-
-[source,shell]
-----
-$ odo service create etcdoperator.v0.9.4 --crd EtcdCluster --dry-run
-----
-
-You can also redirect the output generated above and modify it before starting
-a service. We will see this in <<create-from-yaml,creating service from YAML
-section>>.
-
-=== [[create-service]]Create a service from an Operator 
-
-[NOTE]
-====
-For the commands mentioned in this section to work properly, you need to make
-sure that the Operator has a valid definition in its `metadata` to start the
-requested service. The commands mentioned here refer the
-`metadata.annotations.alm-examples` of an Operator and use it as-is to start
-the service. If this YAML has placeholder values or sample values that are not
-meant to aid in starting a real service, you will not be able to see a service
-start from it.
-====
-
-To start an `EtcdCluster` service from `etcdoperator.v0.9.4` Operator, you need
-to execute:
+In above output, `etcdoperator.v0.9.4` is the Operator while `EtcdCluster`,
+`EtcdBackup` and `EtcdRestore` are the CRDs provided by this Operator.
 
-[source,shell]
-----
-$ odo service create etcdoperator.v0.9.4 --crd EtcdCluster
-----
 
-This is exactly same command as that shown in <<dry-run,printing the YAML of
-service>> section above but without the `--dry-run` flag.
+=== Deploying etcd to a cluster
 
-At the moment, `odo` is unable to list services started from an Operator. To
-check if the above command succeeded in starting a service, use `kubectl` or
-`oc`:
+In this example, we will be deploying our link:https://operatorhub.io/operator/etcd[installed etcd Operator] to an OpenShift / Kubernetes cluster.
 
-[source,shell]
+. Create a etcd service from the `etcdoperator.v0.9.4` Operator:
++
+[source,sh]
 ----
-$ kubectl get EtcdCluster
+  $ odo service create etcdoperator.v0.9.4 --crd EtcdCluster
 ----
 
-At the time of writing this document, above command worked out of the box
-because, as mentioned in the note, `etcdoperator.v0.9.4` 's definition has a
-valid example for `EtcdCluster` embedded into it. This can be checked by doing:
-
-
-[source,shell]
-----
-$ kubectl get csv/etcdoperator.v0.9.4 -o yaml
+. Confirm the Operator was deployed:
++
+[source,sh]
 ----
+  # Kubernetes
+  $ kubectl get EtcdCluster
+  $ kubectl get csv/etcdoperator.v0.9.4 -o yaml
```

Comment:
We don't need this command to verify that the Operator backed service was deployed. It prints the yaml definition of the Operator.